package.json

```diff
@@ -27,35 +27,34 @@
     "url": "https://github.com/swc-project/cli/issues"
   },
   "homepage": "https://github.com/swc-project/cli#readme",
+  "engines": {
+    "node": ">= 12.13"
+  },
   "peerDependencies": {
-    "@swc/core": "^1.2.4"
+    "@swc/core": "^1.2.4",
+    "chokidar": "*"
```

Comment:
Is this okay?
I'm not sure about the wildcard dependency.

Reply:
Sorry, I missed this comment somehow. This is probably harmless left like this but yeah we should probably pin this to v3.